src/components/editor/property/PropertyTemplateOutline.jsx

```diff
@@ -10,25 +10,6 @@ import { isResourceWithValueTemplateRef, resourceToName, templateBoolean } from
 import PropertyComponent from './PropertyComponent'
 import ResourceProperty from './ResourceProperty'
 
-export const addResourceTemplate = (resourceTemplate, reduxPath) => {
```

Comment:
yay!?  getting rid of code.

Reply:
I'd feel better if I understood why it was there to begin with ...